bitnami/thanos/templates/_helpers.tpl

```diff
@@ -255,7 +255,11 @@ Return the queryURL used by Thanos Ruler.
 */}}
 {{- define "thanos.ruler.queryURL" -}}
 {{- if and .Values.queryFrontend.enabled .Values.queryFrontend.ingress.enabled .Values.queryFrontend.ingress.hostname .Values.queryFrontend.ingress.overrideAlertQueryURL -}}
-{{- printf "http://%s" (tpl .Values.queryFrontend.ingress.hostname .) -}}
+  {{- if .Values.queryFrontend.ingress.tls -}}
+    {{- printf "https://%s" (tpl .Values.queryFrontend.ingress.hostname .) -}}
+  {{- else -}}
+    {{- printf "http://%s" (tpl .Values.queryFrontend.ingress.hostname .) -}}
+  {{- end -}}
```

Comment:
```suggestion
  {{- if .Values.queryFrontend.ingress.tls -}}
    {{- printf "https://%s" (tpl .Values.queryFrontend.ingress.hostname .) -}}
  {{- else -}}
    {{- printf "http://%s" (tpl .Values.queryFrontend.ingress.hostname .) -}}
  {{- end -}}
```
```suggestion
    {{ printf "%s://%s" (ternary "https" "http" .Values.queryFrontend.ingress.tls) (tpl .Values.queryFrontend.ingress.hostname .) }}
```

Reply:
done in [`a6a9301` (#24456)](https://github.com/bitnami/charts/pull/24456/commits/a6a93016e06a351205fd0acf5c36386a31f4dab7)